redeem/gcodes/M2x.py

```diff
@@ -297,15 +297,14 @@ def process_gcode(self, g):
     sortby = 'cumulative'
     ps = pstats.Stats(profile, stream=s).sort_stats(sortby)
     ps.print_stats()
-    logging.error(s.getvalue())
+    logging.debug(s.getvalue())
     self.printer.sd_card_manager.reset()
 
   def execute(self, g):
     fn = self.printer.sd_card_manager.get_file_name()
     active = self.printer.sd_card_manager.get_status()
-    logging.info("M24: current file is: '{}'".format(fn))
     if not active:
-      logging.info("M24: active file is '{}'".format(fn))
+      logging.info("M24: Printing file '{}'".format(fn))
```

Comment:
I'd argue that `Selected file` in this instance is more appropriate for logging... As a print is not yet ongoing.

Reply:
No. The next line starts the actual printing and it will be active moments later.

Reply:
Ah. Point.